include/robot_localization/ros_robot_localization_listener.hpp

```diff
@@ -49,6 +49,27 @@
 namespace robot_localization
 {
 
+namespace detail
+{
+rclcpp::SubscriptionOptions
+get_subscription_options_for_qos_override()
```

Comment:
Is a total list of the override options something **not** provided by the rclcpp API? Don't you think this function should be in `rclcpp`?